pkgs/applications/emulators/wine/update-lib.sh

```diff
@@ -0,0 +1,50 @@
+sources_file=pkgs/applications/emulators/wine/sources.nix
+wine_url_base=https://dl.winehq.org/wine
+
+sed_exprs=()
+
+get_source_attr() {
+    nix-instantiate --eval --json -E "(let pkgs = import ./. {}; in pkgs.callPackage $sources_file { inherit pkgs; }).$1" | jq -r
+}
+
+set_source_attr() {
+    path="$1"
+    name="$2"
+    value="$3"
+    line=$(nix-instantiate --eval -E "(builtins.unsafeGetAttrPos \"$name\" (let pkgs = import ./. {}; in pkgs.callPackage $sources_file { inherit pkgs; }).$path).line")
+    sed_exprs+=(-e "${line}s@[^ ].*\$@$name = $value;@")
+}
+
+set_version_and_sha256() {
+    set_source_attr "$1" version "\"$2\""
+    set_source_attr "$1" sha256 "\"$(to_sri "$3")\""
+}
+
+get_latest_wine_version() {
+    list-directory-versions --pname=wine --url="$wine_url_base/source/$1" | grep -v 'diff\|rc\|tar' | sort --reverse --version-sort -u | head -n 1
+}
+
+get_latest_lib_version() {
+    curl -s "$wine_url_base/$1/" | rg -o 'href="[0-9.]*/"' | sed 's_^href="\(.*\)/"_\1_' | sort --reverse --version-sort -u | head -n 1
+}
+
+to_sri() {
+    nix --extra-experimental-features nix-command hash to-sri --type sha256 "$1"
+}
+
```

Comment:
```suggestion
```

We can just take what the nix-daemon outputs

Reply:
Meaning you want to revert the sources to the old hash format? Or is there a way to get `nix-prefetch-url` to output SRIs?

Reply:
Haven't done anything with this because I don't know what direction to go in.

Reply:
nix-prefetch-url is converting it to none sri but thats fine. We don't need to complicate things.